src/group/managed_group/mod.rs

```diff
@@ -105,16 +141,58 @@ impl ManagedGroup {
         credential_bundle: &CredentialBundle,
         members: &[usize],
     ) -> Result<Vec<MLSMessage>, ManagedGroupError> {
-        unimplemented!()
+        let mut plaintext_messages: Vec<MLSPlaintext> = members
+            .iter()
+            .map(|member| {
+                self.group.create_remove_proposal(
+                    &self.aad,
+                    &credential_bundle,
+                    LeafIndex::from(*member),
+                )
+            })
+            .collect();
+
+        // Include pending proposals into Commit
+        let mut messages_to_commit = self.pending_proposals.clone();
```

Comment:
Why are the remove proposals created here are not put in the pending proposal list? I guess I don't understand what the `pending_proposal` list actually holds...